Networking/Networking/Mapper/ProductsBulkUpdateMapper.swift

```diff
@@ -27,21 +31,17 @@ struct ProductsBulkUpdateMapper: Mapper {
 /// This entity allows us to do parse all the things with JSONDecoder.
 ///
 private struct ProductsEnvelope: Decodable {
```

Comment:
nit: simialr to above, `ProductsContainerEnvelope` might be more clear as `ProductsContainer` + data envelope

Reply:
Done in 598e49805a6bc1a2cd50e1d4d3d6c6fca064241c.